keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -82,59 +97,163 @@ been [used by
 sig-storage](https://kubernetes.io/blog/2023/01/02/cross-namespace-data-sources-alpha/)
 to enable cross-namespace data sources.
 
-This KEP proposes moving ReferenceGrant from its current
-`gateway.networking.k8s.io` API group into the `authorization.k8s.io` API
-group.
+This KEP proposes building that overall idea of referential authorization
+directly into Kubernetes. This will build on the original idea of ReferenceGrant
+but add additional capabilities along the way.
+
+## Diagrams
```

Comment:
In case it's too annoying to view these in GitHub, here are some mermaid links:

* [Without RBAC](https://www.mermaidchart.com/raw/f0ec5f4d-b7e3-45b4-bc54-e9c95fa3548f?theme=light&version=v0.1&format=svg) 
* [With RBAC](https://www.mermaidchart.com/raw/268fb055-78ed-4cd6-9bd2-e3f2f7df448b?theme=light&version=v0.1&format=svg)